using_images/xpaas_images/eap.adoc

```diff
@@ -1,119 +1,110 @@
-remote shell session to the running pod:
+. The xPaaS JBoss EAP 6.4 image uses JBoss Web.
+. The xPaas JBoss EAP 7 Beta image uses Undertow instead of JBoss Web. This change only affects users implementing custom JBoss Web Valves in their applications. Affected users must refer to the Red Hat JBoss EAP 7 Beta documentation for details about JBoss EAP Web Valve handlers.
 
-----
-$ oc rsh <pod_name>
-----
+**HornetQ is replaced by Artemis**
 
-Then run the following from the remote shell session to launch the JBoss EAP
-Management CLI:
+. The EAP 6.4 image only uses HornetQ for intra-pod messaging when A-MQ is absent.
+. The EAP 7 Beta image uses Artemis instead of HornetQ. This change resulted in renaming the `*HORNETQ_QUEUES*` and `*HORNETQ_TOPICS*` environment variables to `MQ_QUEUES` and `MQ_TOPICS` respectively. For complete instructions to deal with migrating applications from JBoss EAP 6.4 to 7 Beta, see the JBoss EAP 7 Beta Migration Guide.
+. The EAP 7 Beta image uses Artemis instead of HornetQ. This change resulted in renaming the `*HORNETQ_QUEUES*` and `*HORNETQ_TOPICS*` environment variables to `*MQ_QUEUES*` and `*MQ_TOPICS*` respectively. For complete instructions to deal with migrating applications from JBoss EAP 6.4 to 7 Beta, see the [JBoss EAP 7 Beta Migration Guide].
 
-----
-$ /opt/eap/bin/jboss-cli.sh
-----
+== xPaaS JBoss EAP Version Compatibility Matrix
 
-[WARNING]
-Any configuration changes made using the JBoss EAP Management CLI on a running container will be lost when the container restarts.
+See the xPaaS section of the https://access.redhat.com/articles/2176281[OpenShift and Atomic Platform Tested Integrations page] for details about OpenShift EAP image version compatibility.
 
-link:#Making-Configuration-Changes-EAP[Making configuration changes to the
-JBoss EAP instance inside the JBoss EAP xPaaS image] is different from the process you may be used to for a regular release of JBoss EAP.
+== Set Up the xPaaS JBoss EAP Image
 
-=== Unsupported Configurations
+The following is a list of prerequisites for using the xPaaS JBoss EAP images:
 
-The following is a list of unsupported configurations specific to the JBoss EAP xPaaS image:
+. **Acquire Red Hat Subscriptions** - Ensure that you have the relevant subscriptions for OpenShift as well as a subscription for xPaaS Middleware.
 
-* Using MySQL in a scaled environment with XA distributed transactions is not recommended. For applications that support both scaling and XA distributed transactions, PostgreSQL is recommended instead.
-// This is based on https://issues.jboss.org/browse/CLOUD-56
+. **Install OpenShift** - Before using the xPaaS JBoss EAP images, you must have an OpenShift environment installed and configured:
 
-ifdef::openshift-enterprise[]
-== Using the JBoss EAP xPaaS Image Streams and Application Templates
+.. The link:../install_config/install/quick_install.html method allows you to install OpenShift using an interactive CLI utility.
```

Comment:
Tested locally and I did miss out on a level. tested and substituted all link:../ paths with link:../../ paths.